catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -248,7 +249,13 @@ public void setProcessingMechanism(String processingMechanism) {
      * @param attributeOverrides - a list of attributes to override
      */
     public void setAttributeOverrides(List<String> attributeOverrides) {
-        this.attributeOverrides = attributeOverrides;
+        Map<String, Serializable> attributeOverrideMap = new HashMap<>();
+        for (String s : attributeOverrides) {
+            String[] keyValue = s.split("=");
+            attributeOverrideMap.put(keyValue[0], keyValue[1]);
```

Comment:
I think this approach may break multivalued attributes. The documentation currently states that multiple override values are specified by repeating the `key=value` entries, so we would need to collect those values somehow. Perhaps a map of lists.